ceno_emul/src/platform.rs

```diff
@@ -9,33 +9,42 @@ use crate::addr::{Addr, RegIdx};
 /// - codes of environment calls.
 #[derive(Clone, Debug)]
 pub struct Platform {
-    pub rom: Range<Addr>,
-    pub ram: Range<Addr>,
+    pub prog_code: Range<Addr>,
```

Comment:
We don't need `prog_code` in the platform.  There's no restrictions we need to enforce, apart from what's in the ELF, and we enforce them during decoding.

That's all thanks to the recent move to a Harvard Architecture.

Reply:
Hmm, all the comments here for this file seem to have moved down by one line, compared to where they should be.